CHANGELOG.md

```diff
@@ -1,15 +1,70 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
+# [v1.7.0](unreleased)
+### Notes
+- This release supports API2200 minimally where we can use OneView API2400 with this SDK and ImageStreamer REST API version 2000 (I3S v5.40).  
```

Comment:
Change to "supports API2400 minimally" and i3s also

Reply:
Done